packages/training/deploy/docker/Dockerfile.bench

```
@@ -0,0 +1,141 @@
+#     -e HF_TOKEN=hf_xxx \
+#     babylon-benchmark:latest
+#
+# =============================================================================
+
+# -----------------------------------------------------------------------------
+# Stage 1: vLLM Base (same as training)
+# -----------------------------------------------------------------------------
+FROM vllm/vllm-openai:v0.14.0 AS vllm-base
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)